### ISSUE TITLE
Access to the training accuracy 

### ISSUE THREAD

**Issue by BBELAOUCHA** [automl/auto-sklearn#670]
Hi, 
I want to compute the training accuracy so that I can compare it with the validation accuracy to have an idea about the over fitting of the the model. 
Does it make sense or because it is an ensemble that uses different models we cant use the two scores mentioned earlier to decide weather the ensemble suffers from bias.

I checked in the ensemble, there is .ensemble_.train_score_, does it contain the training score of the last cv fold which is not the same as the training score corresponding to the best  cv score. Am I right?

Thx in advance

**Comment by tsinggggg** [comment]
i think training loss could be found in tmp folder

**Comment by mfeurer** [comment]
We finally have this in an example: https://automl.github.io/auto-sklearn/master/examples/40_advanced/example_get_pipeline_components.html

The example is so far only available in the documentation for the development branch, but also works on the master branch.
